chunky/src/java/se/llbit/chunky/renderer/Postprocess.java

```diff
@@ -36,6 +36,12 @@
     @Override public String toString() {
       return "Tonemap operator 1";
     }
+  },
+  TONEMAP2 {
+    @Override public String toString() { return "ACES filmic tone mapping"; }
```

Comment:
I have no preference about annotation placements - I usually have them on the same line if it fits.